src/main/java/org/alfresco/repo/solr/NodeParameters.java

```diff
@@ -181,15 +182,25 @@ public void setIncludeAspects(Set<QName> includeAspects)
     public void setExcludeAspects(Set<QName> excludeAspects)
     {
         this.excludeAspects = excludeAspects;
-    }
-
-    public String getShardProperty()
-    {
-        return this.shardProperty;
-    }
-
-    public void setShardProperty(String shardProperty)
-    {
-        this.shardProperty = shardProperty;
     }
+
+    public String getShardProperty()
+    {
+        return this.shardProperty;
+    }
+
+    public void setShardProperty(String shardProperty)
+    {
+        this.shardProperty = shardProperty;
+    }
+
+    public String getCoreName() {
+        return this.coreName;
+    }
+
+    public void setCoreName(String coreName){
+        this.coreName = coreName;
+    }
+

```

Comment:
Only including "getCoreName" and "setCoreName" methods, but changes are affecting other parts of the code.